project/frontend/src/pages/HomePage.js

```diff
@@ -20,20 +20,29 @@ function HomePage(props) {
     setLocationName(locationData.locationName);
   };
 
+  const buttonStyle = {
+    opacity: locationName ? 1 : 0,
+    transform: locationName ? 'translateY(0px)' : 'translateY(-24px)',
+  };
+
```

Comment:
optional: if more stuff gets added to this, would it be cleaner to have 
```
const buttonStyle = locationName ? { opacity: 1, transform: 0px} : { opacity: 0, transform: -24px} 
```
(if thats a syntax thats valid)

Reply:
Tbh I don't anticipate any more styles being added here since I already have a class on these buttons that sets their constant styles. I'll swap it out for your recommendation though because I do think the one ternary is cleaner than two. Thanks!